corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/controller.js

```diff
@@ -5,11 +5,17 @@ hqDefine("cloudcare/js/formplayer/apps/controller", function () {
         FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
         settingsViews = hqImport("cloudcare/js/formplayer/layout/views/settings"),
         views = hqImport("cloudcare/js/formplayer/apps/views");
+
     return {
         listApps: function () {
-            $.when(FormplayerFrontend.getChannel().request("appselect:apps")).done(function (apps) {
+            $.when(FormplayerFrontend.getChannel().request("appselect:apps")).done(function (appCollection) {
+                var apps = appCollection.toJSON();
```

Comment:
Does this one need to be a `var` rather than a `let`?

Reply:
Are you referring to `apps`? I think I should actually change it to `let` since it's only used within the block